ios/brave-ios/Sources/BraveWallet/Crypto/Transactions/TransactionParser.swift

```diff
@@ -197,6 +216,55 @@ enum TransactionParser {
             )
           )
         )
+      } else if let btcTxData = transaction.txDataUnion.btcTxData {  // BTC send tx
+        // Require 8 decimals precision for BTC parsing
+        let formatter = WeiFormatter(decimalFormatStyle: .decimals(precision: 8))
+        let namedFromAccount = fromAccountInfo.name
+        let fromValue = "\(btcTxData.amount)"
+        let fromValueFormatted =
+          formatter.decimalString(
+            for: fromValue,
+            radix: .decimal,
+            decimals: Int(network.decimals)
+          )?.trimmingTrailingZeros ?? ""
+        let fromFiat =
+          currencyFormatter.string(
+            from: NSNumber(
+              value: assetRatios[network.nativeToken.assetRatioId.lowercased(), default: 0]
+                * (Double(fromValueFormatted) ?? 0)
+            )
+          ) ?? "$0.00"
```

Comment:
Same question

Reply:
This one is tricky because we already have a default to 0 so this should never be hit unless the `NumberFormatter` is failing to generate a currency formatted string for 0 value. We could use `CurrencyCode.symbol` but it'd be hardcoded to left-side for all languages without custom mapping, but again shouldn't be hit 🤔.